.gitignore

```diff
@@ -433,5 +433,7 @@ FodyWeavers.xsd
 ### END VisualStudio.gitignore ###
 ##################################
 
+*.wixpdb
```

Comment:
I suppose it would be becoming to add a comment above for a Wix-specific section